src/linalg/spectral.jl

```diff
@@ -79,8 +80,9 @@ function laplacian_matrix(g::AbstractGraph{U}, T::DataType=Int; dir::Symbol=:uns
         dir = is_directed(g) ? :both : :out
     end
     A = adjacency_matrix(g, T; dir=dir)
-    D = convert(SparseMatrixCSC{T,U}, Diagonal(sparse(sum(A, dims=2)[:])))
-    return D - A
+    s = sum(A;dims=2)
```

Comment:
leave a space between arguments and after `;` (I know the previous version didn't do this, but we're making a concerted effort here.)

See L84 as well.